app/views/devise/registrations/new.html.erb

```diff
@@ -0,0 +1,33 @@
+<div  class="form-heading">
+  <h2>Sign up</h2>
+  <hr>
+</div>
+<div class="form-group">
+  <%= form_for(resource, as: resource_name, url: registration_path(resource_name)) do |f| %>
+    <%= render "devise/shared/error_messages", resource: resource %>
+    <div class="field">
+      <%= f.text_field :name, placeholder: "User name", autofocus: true %>
+    </div>
+    <div class="field">
+      <%= f.text_field :photo, placeholder: "Photo url"%>
+    </div>
+    <div class="field">
+      <%= f.email_field :email, autocomplete: "email", placeholder: "Email" %>
+    </div>
+    <div class="field">
+      <%= f.password_field :password, placeholder: "password (6 characters minimum)" %>
+    </div>
+    <div class="field">
+      <%= f.password_field :password_confirmation, placeholder: "password confirmation" %>
+    </div>
+    <div class="field">
+      <%= f.text_area :bio, placeholder:"Brief bio about yourself..." %>
+    </div>
+    <div class="actions">
+      <%= f.submit "Sign up" %>
+    </div>
+    <div class="extras">
+      <%= render "devise/shared/links" %>
+    </div>
+  <% end %>
+</div>
```

Comment:
- [ ] Hi, according to the requirements this page should match the specified [wireframe](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_login.png) 

|Wireframe Image| Your Image|
-|-
|![image](https://user-images.githubusercontent.com/18730106/162581060-6585e873-85b9-4a8e-a7b3-09603fca7801.png)|![image](https://user-images.githubusercontent.com/18730106/162581033-19209316-01aa-4a49-ad4f-f0c841d9550c.png)

Reply:
Hi @MrBrN197  thank you for taking the time and review my app
here is my implementation 
in project requirements, I asked to make signup, forgot my password, and a confirmation email which is not founded in the wireframe so I suggested that here is a perfect place to put them
![Screenshot from 2022-04-09 18-08-30](https://user-images.githubusercontent.com/51102661/162582069-7437b17f-bf8d-46e3-8d2b-7d2fd1025153.png)

Reply:
@ReAliens  Hi, can I ask where the page you have linked is? currently, when I go to your sign_out page this is not how it looks

Reply:
@MrBrN197 root page 
i didn't implement sig out page as it's not required

Reply:
this page on http://localhost:3000/

Reply:
Sorry I mean log in page. When I go to your root page. I simply see a blank page.